frontend/src/metabase/redux/downloads.ts

```diff
@@ -143,11 +146,13 @@ const getDatasetParams = ({
     // For whatever wacky reason the /api/embed endpoint expect params like ?key=value instead
     // of like ?params=<json-encoded-params-array> like the other endpoints do.
```

Comment:
Might be worth updating this comment, as now the endpoints will work with json encoded params